cmd/bee/cmd/cmd.go

```diff
@@ -44,6 +44,10 @@ const (
 	optionNameGatewayMode          = "gateway-mode"
 	optionNameClefSignerEnable     = "clef-signer-enable"
 	optionNameClefSignerEndpoint   = "clef-signer-endpoint"
+	optionNameSwapEndpoint         = "swap-endpoint"
+	optionNameSwapFactoryAddress   = "swap-factory-address"
+	optionNameSwapInitialDeposit   = "swap-initial-deposit"
+	optionNameSwapEnable           = "swap-enable"
```

Comment:
Let's work together with @vandot to remove this flag asap.

Reply:
we might want to keep this around for now and just default to true. otherwise every dev will have to spawn local blockchains with factories and token prefundings just to test completely unrelated stuff.

Reply:
Agree.